src/server/game/Entities/Unit/Unit.cpp

```diff
@@ -501,6 +501,19 @@ void Unit::Heartbeat()
 
     // SMSG_FLIGHT_SPLINE_SYNC for cyclic splines
     SendFlightSplineSyncUpdate();
+
+    // Aura Heartbeat
+    TriggerAuraHeartbeat();
+}
+
+void Unit::TriggerAuraHeartbeat()
+{
+    // There are some auras which do invoke generic behavior on heartbeat
+    for (auto const& [_, auraApplication] : m_appliedAuras)
+        auraApplication->GetBase()->Heartbeat();
+
+    // PROC_FLAG_HEARTBEAT
```

Comment:
is its rly usefull comment?

Reply:
it's called consistency and making sure that if someone ever starts messing around with it, that something remains to tell what all this was suposed to mean.

Reply:
is possible to understand it from args .... Unit::ProcSkillsAndAuras(.....PROC_FLAG_HEARTBEAT